source/MRViewer/MRRibbonMenuSearch.cpp

```diff
@@ -204,6 +204,7 @@ void RibbonMenuSearch::drawWindow_( const Parameters& params )
             params.btnDrawer.drawButtonItem( *foundItem.item, dbParams );
             if ( foundItem.item->item->isActive() != pluginActive )
             {
+                pushRecentItem( foundItem.item->item );
```

Comment:
will it work for `RibbonMenuItem` (looks like it only works for `StatePlugin` now)

Reply:
fixed